NEWS.md

```diff
@@ -0,0 +1,3141 @@
+9.4.0 Release Notes
+-------------------
+
+Updates
+-------
+
```

Comment:
I think it would be better to use the standard markdown headings
```suggestion
# 9.4.0 Release Notes

## Updates

```

Without the indents and the same symbols used for denoting the heading it is not really clear that "Updates" is a subsection of "9.4.0 Release Notes".

Applies across the whole document.

Reply:
Done. I hope I made all of them

Reply:
Cool! that way you can link to an specific section, like https://github.com/jjimenezshaw/PROJ/blob/md-files/NEWS.md#700-release-notes